tests/Conversations/ConversationFiltersTest.php

```diff
@@ -44,6 +45,7 @@ public function testGetParams()
             'sortField' => 'createdAt',
             'sortOrder' => 'asc',
             'query' => 'query',
+            'embed' => 'threads',
```

Comment:
Thanks so much for taking the time to kick this off.

Our terminology `with*()` in the filters makes things a bit confusing.  I wish we had changed this prior to the 2.* launch, but unfortunately it'd be a breaking change at this point.  `*Filter` classes are used to narrow down the results and `*Request` objects are used to eager load objects.  I think in this case, we'd need `ConversationRequest.withThreads()` to be the initiator of eager loading the threads rather than `withEmbed()`.  That'd also ensure backwards compatibility.